process/io/plot_radial_build.py

```diff
@@ -460,7 +461,7 @@ def main(args=None):
     plt.xlabel("Radius [m]")
     plt.tight_layout()
     plt.savefig(
-        f"{args.outputdir}/{args.input[:-10]}_radial_build.{save_format}",
+        f"{args.outputdir}/{os.path.splitext(os.path.basename(args.input))[0]}_radial_build.{save_format}",
```

Comment:
```suggestion
        f"{args.outputdir}/{Path(args.input).stem}_radial_build.{save_format}",
```

This is probably the correct and less-verbose way to do this in newer versions of Python (of course this would also make the `import os` redundant.

Reply:
Yes, you're right. Fixed

Reply:
Thank you very much :)